components/search_results/search_results.jsx

```diff
@@ -225,6 +225,16 @@ export default class SearchResults extends React.PureComponent {
                         {ctls}
                     </div>
                 </Scrollbars>
+                <div
+                    className='loading-screen'
+                    style={this.props.isSearchGettingMore ? {} : {display: 'none'}}
```

Comment:
Instead of just hiding this div when `isSearchGettingMore` is false, can we just not mount it in that case?

Reply:
@hmhealey for consistency as well as (minor) performance, that makes sense. I've refactored.